### ISSUE TITLE
go get github.com/cweill/gotests/...: malformed module path "github.com/cweill/gotests/...": double dot

### ISSUE THREAD

**Issue by liulangwa** [cweill/gotests#119]
## env
go version go1.12.14 windows/amd64

set GOARCH=amd64
set GOBIN=
set GOCACHE=xxxx
set GOEXE=.exe
set GOFLAGS=
set GOHOSTARCH=amd64
set GOHOSTOS=windows
set GOOS=windows
set GOPATH=E:\work\goworkspace
set GOPROXY=https://goproxy.io
set GORACE=
set GOROOT=c:\go
set GOTMPDIR=
set GOTOOLDIR=c:\go\pkg\tool\windows_amd64
set GCCGO=gccgo
set CC=gcc
set CXX=g++
set CGO_ENABLED=1
set GOMOD=NUL
set CGO_CFLAGS=-g -O2
set CGO_CPPFLAGS=
set CGO_CXXFLAGS=-g -O2
set CGO_FFLAGS=-g -O2
set CGO_LDFLAGS=-g -O2
set PKG_CONFIG=pkg-config
set GOGCCFLAGS=-m64 -mthreads -fmessage-length=0 -fdebug-prefix-map=C:\Users\DUYONG~1\AppData\Local\Temp\go-build265818502=/tmp/go-build -gno-record-gcc-switches
## go get failed 

```
go get github.com/cweill/gotests/...: malformed module path "github.com/cweill/gotests/...": double dot
```

**Comment by shihanng** [comment]
This appears to be a known issue in Go 1.12: https://github.com/golang/go/issues/29363
and it is fixed for Go 1.13.

For Go 1.12, the work around to install the CLI is

```
go get -u github.com/cweill/gotests/gotests
```

**Comment by liulangwa** [comment]
thinks ,go 1.13 fixed this problem
